Cargo.toml

```diff
@@ -66,6 +67,7 @@ dora-coordinator = { version = "0.3.5", path = "binaries/coordinator" }
 dora-ros2-bridge = { path = "libraries/extensions/ros2-bridge" }
 dora-ros2-bridge-msg-gen = { path = "libraries/extensions/ros2-bridge/msg-gen" }
 dora-ros2-bridge-python = { path = "libraries/extensions/ros2-bridge/python" }
+# versioned independently from the other dora crates
 dora-message = { version = "0.3.5", path = "libraries/message" }
```

Comment:
Should we start dora-message from "0.0.1" as it might be confusing to use `0.3.5` on a `0.3.6` version of dora.

Reply:
We already published the `dora-message` crate to crates.io, so we have to use at least version 0.3.6. My preferred solution would be to bump the version to 0.4.0 to make it clear that this version number is unrelated to the main dora version. It also marks a clear cutoff of the versioning approach, i.e. old versioning strategy for versions 0.3 (and before) and new versioning strategy for >=0.4.

I'll push a commit for that.